app/controllers/groups_wallets_controller.rb

```diff
@@ -0,0 +1,29 @@
+class GroupsWalletsController < ApplicationController
+  before_action :authenticate_user!
+  def new
+    @wallet = Wallet.new
+    @groups = current_user.groups
+    @back_id = params[:id]
+  end
+
+  def create
+    expense = Wallet.create(name: wallet_params[:name], author_id: current_user.id, amount: wallet_params[:amount])
+
```

Comment:
- [OPTIONAL] Great job 👏🏻  However I would suggest the use of [Strong Parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters) example:

```ruby
  def create
    Person.create(params[:person])
  end
```